test/functional/Jsr292/src/com/ibm/j9/jsr292/LookupAPITests_Find.java

```diff
@@ -2098,6 +2099,35 @@ public void test_FindSpecial_Public_CrossPackage_InnerClass() throws Throwable {
 		AssertJUnit.assertEquals( 3, out );
 	}
 	
+	/**
+	 * findSpecial test using a default method of an interface where the class (the resultant 
+	 * method handle will be bound to) is in a different package.
+	 * @throws Throwable
+	 */
+	@Test(groups = { "level.extended" })
+	public void test_FindSpecial_Default_CrossPackage_Interface() throws Throwable {
+		MethodHandle example = MethodHandles.lookup().findSpecial(CrossPackageDefaultMethodInterface.class, "addDefault", MethodType.methodType(int.class, int.class, int.class), CrossPackageDefaultMethodInterface.class);
+	}
+	
+	/**
+	 * Negative findSpecial test using a public method of a class where the class (the resultant 
+	 * method handle will be bound to) is in a different package.
+	 * Note: when the lookup class is not equal to the caller class, the declaring class
+	 * must be an interface which the caller class can be assigned to.
+	 * @throws Throwable
+	 */
+	@Test(groups = { "level.extended" })
+	public void test_FindSpecial_Public_CrossPackage_Not_Interface() throws Throwable {
+		boolean illegalAccessExceptionThrown = false;
+		try {
+			MethodHandles.lookup().findSpecial(PackageExamples.class, "addPublic", MethodType.methodType(int.class, int.class, int.class), PackageExamples.class);
```

Comment:
The preferred pattern for this kind of test is to use a `fail` method after the code that's expected to throw the exception like:
https://github.com/eclipse/openj9/blob/b8cdef32c53c4c2fd1e427e1692a49ac8c4d5634/test/functional/Jsr292/src/com/ibm/j9/jsr292/LookupAPITests_Find.java#L375-L378

Reply:
Updated the pattern as suggested.